test/schema/basalTest.js

```diff
@@ -242,7 +241,7 @@ describe('schema/basal.js', function(){
     });
 
     describe('scheduleName', function(){
-      helper.rejectIfAbsent(goodObject, 'scheduleName');
+      helper.okIfAbsent(goodObject, 'scheduleName');
```

Comment:
We don't have `scheduleName` yet for Tandem, and it turns out we can't get it for OmniPod. We're providing it now, but it's a placeholder and a lie... We also won't be able to get it for Animas.